FuncGameDev/FSharp/Services/ScoreSavingService.fs

```diff
@@ -2,18 +2,23 @@
 
 open System.IO
 
-let getScore = 
+
+// Scores is an array of Strings. 
+// Scores [0] represents the floor reached
+// Scores [1] represents the time remaining
+// Scores [2] represents the total time spent
+let getScores = 
     let baseDirectory = __SOURCE_DIRECTORY__
     let baseDirectory' = Directory.GetParent(baseDirectory)
     let filePath = "HighScore.txt"
     let fullPath = Path.Combine(baseDirectory'.FullName, filePath)
-    let score =  File.ReadAllText(fullPath)
-    score |> int
+    let scores =  File.ReadAllLines(fullPath)
+    scores
 
-let storeScore score =
+let storeScores scores =
     let baseDirectory = __SOURCE_DIRECTORY__
     let baseDirectory' = Directory.GetParent(baseDirectory)
     let filePath = "HighScore.txt"
     let fullPath = Path.Combine(baseDirectory'.FullName, filePath)
-    File.WriteAllText(fullPath, score)
+    File.WriteAllLines(fullPath, scores)
```

Comment:
It seems right now we're always writing the score. This makes it more of a "recent scores" file than a "high scores" one.